packages/database/src/migrations/20210610002626-RewriteTotalTestsPerCapitaToUseIndicator-modifies-data.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+import { generateId, insertObject } from '../utilities';
+
+const updateDashboard = async (db, dashboardId, dataBuilder, dataBuilderConfig) => {
+  await db.runSql(`
+    UPDATE "dashboardReport" 
+    SET "dataBuilder" = '${dataBuilder}'
+    where id = '${dashboardId}';
+  `);
+
+  await db.runSql(`
+    UPDATE "dashboardReport" 
+    SET "dataBuilderConfig" = '${JSON.stringify(dataBuilderConfig)}'
+    where id = '${dashboardId}';
+  `);
+};
+
+const insertIndicator = async (db, indicator) => {
+  await insertObject(db, 'indicator', { ...indicator, id: generateId() });
+  await insertObject(db, 'data_source', {
+    id: generateId(),
+    code: indicator.code,
+    type: 'dataElement',
+    service_type: 'indicator',
+  });
+};
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const DASHBOARD_ID = 'COVID_Tests_Per_Capita';
+
+const DAILY_TESTS_PER_CAPITA_INDICATOR = {
+  code: 'COVID_AU_Daily_Tests_Per_Capita',
+  builder: 'analyticArithmetic',
+  config: {
+    formula: 'round((dailysurvey006  * 100000) / AU_POP002)',
+    aggregation: {
+      dailysurvey006: 'FINAL_EACH_DAY',
+      AU_POP002: 'FINAL_EACH_DAY_FILL_EMPTY_DAYS',
+    },
+  },
+};
+
+const DATA_BUILDER_AND_CONFIG = {
+  oldConfig: {
+    dataBuilder: 'sumPreviousValuesPer100kPerDayByOrgUnit',
```

Comment:
Using an indicator is so much better than having to maintain a specific `sumPreviousValuesPer100kPerDayByOrgUnit` data builder!